content/docs/09.administrator-guide/01.configuration/04.others.md

```diff
@@ -280,3 +280,21 @@ kestra:
         }
       </style>
 ```
+
+
+## JDBC Healthcheck
+
+JDBC have a healthcheck that will check every worker at every `frequency`.
+
+* If the last healthcheck is older than `heartbeat-missed * frequency`, the worker will be marked as DEAD.
```

Comment:
```suggestion
* If the last healthcheck is older than `heartbeat-missed * frequency`, the worker will be marked as unhealthy.
```
is it necessary to say DEAD?

Reply:
Well, it is the exact status, but unhealthy works too I guess